help/license/reuse.md

```diff
@@ -0,0 +1,49 @@
+Reuse Requests
+==============
+
+This FAQ is an attempt to collect answers to your common questions surrounding reusing content from arXiv in your materials. 
+
+- [Can I reuse figures from an arXiv paper?](#figures) 
+- [Do I need arXiv's permission to repost the full text?](#full_text)
+- [How can I determine what license the version was assigned?](#what_license)
+- [I want to include a paper of mine from arXiv in my thesis, do I need specific permission?](#thesis)
+- [I want to include a paper of mine from arXiv in an Institutional Repository, do I need permission?](#ir)
```

Comment:
Institutional repository should probably be lower case, unless there is an institutional repository named, "Institutional Repository".